Services/DataBaseService/MigrationScripts/03.2020/03_UserTableCreate.sql

```diff
@@ -0,0 +1,22 @@
+USE [TradingStation];
+
+IF OBJECT_ID('dbo.Users', 'U') IS NOT NULL
+    DROP TABLE [dbo].[Users]
+
+CREATE TABLE [dbo].[Users]
+    (
+        Id              uniqueidentifier    NOT NULL,
+        Email           NVARCHAR(40)        NOT NULL,
+        PasswordHash    NVARCHAR(MAX)       NOT NULL,
+        FirstName       NVARCHAR(15)        NOT NULL,
+        LastName        NVARCHAR(20)        NULL,
```

Comment:
User with only first name?

Reply:
> User with only first name?
Well, why not? As long as we don't check passports and work in the sandbox, it may be some "Mustapha with no lastname". However, I can mark it `NOT NULL` if needed